### ISSUE TITLE
[FR] Add enpoints to the web front to query basic informations about a player

### ISSUE THREAD

**Issue by xerxes-at** [RaidMax/IW4M-Admin#61]
Add a simple API to the web front to query basic information about a player via his GUID.
One for privileges with: 
* Admin access
* Player group
* is fully Authed (eg still needs to log in)

And one with the last 3 (or variable) penalties.

Stuff that could be useful for others:
* connection count
* country

This API comes handy for client sided mods that need to know about the players permissions. Additionally a dvar (eg `_IW4MA`) should be set to the address of the web interface when the bot comes online, obviously there should be a setting to overwrite the address in case it's running behind a reverse proxy.

**Comment by Bluscream** [comment]
Why was this closed?

**Comment by RaidMax** [comment]
It was implemented some time ago. See https://github.com/RaidMax/IW4M-Admin/blob/master/Plugins/IW4ScriptCommands/GscApiController.cs


________________________________

**Comment by xerxes-at** [comment]
I asked RaidMax to close this request, for the reason stated above.
I already have the first 3 mini scripts using it on `138.201.26.173:28960` and `138.201.26.173:28990`.
The only minor disadvantage in the current implementation is that the bot needs to already have pulled the data from the DB because otherwise you will get an empty result back. I wrote a small collection of methods and functions to handle all communication between iw4ma and my gsc scripts.

Something strange I noticed regarding the iw4x gsc http interface is that it requests everything twice (but only reads it once from the server).
